tls/s2n_prf.c

```diff
@@ -268,6 +268,9 @@ static int s2n_p_hash(struct s2n_prf_working_space *ws, s2n_hmac_algorithm alg,
     if (seed_b) {
         GUARD(hmac->update(ws, seed_b->data, seed_b->size));
     }
+    if (seed_c) {
```

Comment:
Should it be possible to supply a seed_c without a seed_b (i.e., is it correct to nest these if clauses instead of having them be sequential)?

Reply:
I decided to nest them and added two checks for correct seed usage.